src/native/libs/System.IO.Compression.Native/zlib_allocator_win.c

```diff
@@ -92,7 +91,7 @@ typedef struct _DOTNET_ALLOC_COOKIE
 const SIZE_T DOTNET_ALLOC_HEADER_COOKIE_SIZE_WITH_PADDING = (sizeof(DOTNET_ALLOC_COOKIE) + MEMORY_ALLOCATION_ALIGNMENT - 1) & ~((SIZE_T)MEMORY_ALLOCATION_ALIGNMENT  - 1);
 const SIZE_T DOTNET_ALLOC_TRAILER_COOKIE_SIZE = sizeof(DOTNET_ALLOC_COOKIE);
 
-voidpf ZLIB_INTERNAL zcalloc(opaque, items, size)
+voidpf z_custom_calloc(opaque, items, size)
```

Comment:
z_custom_calloc and z_custom_free are what zlib-ng call for allocations, hence the necessary function renaming.

Reply:
My memory's failing me. Let me check the chat. I thought this was the last thing we decided.

Reply:
Searching through the code, we specifically changed the name to avoid symbol conflicts (and we are setting them on the `ZStream*` object).

Reply:
I remember what happened: I renamed it to `*_custom_*` to confirm it was working. But I forgot to revert to the original name.

I still need to remove `ZLIB_INTERNAL` though.

Reply:
Ok clarified. My first comment was wrong. zlib-ng is not calling the custom methods, we are overriding the ones zlib-ng uses with these custom methods.